hr_timesheet_sheet_no_negative/views/res_config_settings_views.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+
+    <record id="res_config_settings_view_form" model="ir.ui.view">
+        <field name="name">res.config.settings.view.form.inherit.hr.timesheet.no.negative</field>
+        <field name="model">res.config.settings</field>
+        <field name="inherit_id" ref="hr_timesheet_sheet.res_config_settings_view_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//div[@name='timesheet_week_start']" position="after">
+                <div class="col-xs-12 col-md-6 o_setting_box">
+                    <div class="o_setting_right_pane">
+                        <label for="timesheet_negative_unit_amount"/>
```

Comment:
Shoudn't this have an icon about being company-specific?

Reply:
Can you link me an example?

Reply:
https://github.com/OCA/hr-timesheet/blob/12.0/hr_timesheet_role/views/res_config_settings.xml#L21

Reply:
thanks